src/test/java/org/cactoos/scalar/LengthOfTest.java

```diff
@@ -185,4 +186,15 @@ public void lengthOfEmptyIterator() {
             new IsEqual<>(0)
         ).affirm();
     }
+
+    @Test
+    public void lengthOfText() {
+        new Assertion<>(
+            "Must calculate length of empty iterator",
```

Comment:
@andreoss please, fix this message, because the string isn't empty.

Reply:
@fabriciofx Fixed